trustpinning/certs_test.go

```diff
@@ -1007,12 +1003,79 @@ func generateExpiredTestingCertificate(rootKey data.PrivateKey, gun string) (*x5
 	return cryptoservice.GenerateCertificate(rootKey, gun, startTime, startTime.AddDate(1, 0, 0))
 }
 
+// Helper function for explicitly generating key IDs and unexported fields for equality testing
+func generateRootKeyIDs(r *data.SignedRoot) {
```

Comment:
I'm not sure if this function is still used anywhere - should it be removed?  Although you do bring up a good point;  it would be good to useful to have equality test functions for some of our data structures that have unexported fields at some point.